### ISSUE TITLE
Error using the BioLink model

### ISSUE THREAD

**Issue by vemonet** [monarch-initiative/ontogpt#22]
Hi, I tried to use the BioLink model as base to run the SPIRES engine (as it was recommended in the readme!), but faced some issue. 

Generating the `.py` file from the `biolink_model.yaml` put in `src/ontogpt/templates/` worked well with `make`

But we encounter an error when trying to extract a class:

```
poetry run ontogpt extract -t biolink_model.ChemicalToDiseaseOrPhenotypicFeatureAssociation treatment.txt
```

We are getting the error `ValueError: Template biolink_model.ChemicalToDiseaseOrPhenotypicFeatureAssociation not found` because the classes name defined in the BioLink model are in the format `chemical to disease or phenotypic feature association`

If we try to provide the class name with spaces:

```
poetry run ontogpt -v extract -t "biolink_model.chemical to disease or phenotypic feature association" treatment.txt
```

There is an error due to the python classes name not matching:

```
  File "/home/vemonet/develop/translator/ontogpt/src/ontogpt/engines/knowledge_engine.py", line 230, in _get_template_class
    self.template_pyclass = mod.__dict__[class_name]
KeyError: 'chemical to disease or phenotypic feature association'
```

I could easily implement a fix I think, but I wonder if anyone here with a better knowledge of LinkML than me has a quick fix for this problem? @cmungall

**Comment by vemonet** [comment]
A better solution would be to fix BioLink at the source so that it is compliant with LinkML

So that in the future we can build tools that are working with LinkML, and use it also with BioLink. 

So we don't need to add tons of hacks in each tool we build. Because on the long run adding all those hacks will prevent to actually build and maintained tools.

Interoperability is really important (imo), I thought this was one of the goal of LinkML, but it's gone if we need to add specific hacks just because of some spaces in labels

**Comment by nlharris** [comment]
Just wondering if this has been fixed yet.

**Comment by caufieldjh** [comment]
No, not yet - the workaround is to replace the offending class name with something equivalent to the Python class, like `chemical to disease or phenotypic feature association` -> `ChemicalToDiseaseOrPhenotypicFeatureAssociation`.
But ideally that shouldn't be necessary.
